src/index.js

```diff
@@ -210,6 +210,34 @@ export const rules = {
       );
     },
   },
+  'type-names-cap': {
```

Comment:
@DianaSuvorova this rule name could be more descriptive, especially for folks who don't know what `cap` is short for. what would you think about `capitalized-type-name`? (inspired by [capitalized-comments](http://eslint.org/docs/rules/capitalized-comments) from eslint)